app/javascript/packs/containers/NavigationBar/NavigationBarContainer.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import {loadInitialState} from '../../actions/initialState';
+import {getBuildings, getNavBarBuildings} from '../../selectors/buildingsSelector';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import DropdownMenuContainer from './DropdownMenuContainer'
+import {getName, getEmail} from '../../selectors/usersSelector'
+import {getCurrentCategory, getCategories} from "../../selectors/categoriesSelector";
+import CategoryContainer from './CategoryContainer';
+import {getQuestionsByBuildingNavBar, getQuestionsByCategory} from "../../selectors/questionsSelector";
+import {getRemainingAnswersforCategory} from "../../selectors/answersSelector";
+
+
+class NavigationBarContainer extends React.Component {
+    componentDidMount() {
+        if (window.INITIAL_STATE) {
+            this.props.initActions.loadInitialState(window.INITIAL_STATE);
```

Comment:
Should this component need to call the `loadInitialState` action? We actually do that in the `RootContainer` component and it should not need to be called again.